CMakeLists.txt

```diff
@@ -88,6 +88,12 @@ if(ENABLE_ERM AND NOT ENABLE_LUA)
 	set(ENABLE_LUA ON)
 endif()
 
+if(ENABLE_LAUNCHER)
+	option(ENABLE_INNOEXTRACT "Enable innoextract for GOG file extraction in launcher" ON)
+else()
+	set(ENABLE_INNOEXTRACT OFF)
+endif()
+
```

Comment:
Not sure if CMake provides any better way to handle such cases, but this might be a little inconvenient:
- disable launcher
- run cmake, do stuff
- re-enable launcher, run cmake

You now have innoextract silently disabled even though this was not your intention and you never touched that option

Feel free to ignore if you can't think of better option(s)

Reply:
maybe dependent option would help https://cmake.org/cmake/help/latest/module/CMakeDependentOption.html#module:CMakeDependentOption